enatega-multivendor-admin/src/components/Restaurant/CreateRestaurant.jsx

```diff
@@ -262,7 +272,7 @@ const CreateRestaurant = props => {
 
 
   return (
-    <Box container className={classes.container}>
+   <Box container className={classes.container} onClick={handleContainerClick}>
```

Comment:
For disabling modal to close when click outside, use material UI's modal props "disableBackdropClick".